### DISCUSSION TITLE
.NET MAUI app full screen issue

### DISCUSSION THREAD

**Discussion by lydiasoftnz** [community/community#136257] [Mobile]
### Select Topic Area

Question

### Body

When I create a .NET MAUI app and run it in landscape mode, a persistent black line appears on the right side of the screen. I’ve attempted to enforce full-screen mode, but the black line still remains.

![image](https://github.com/user-attachments/assets/f0558048-b7a8-4f95-8fd6-e99a34cb1edf)

**Comment by Akash1134** [comment]
Hi @lydiasoftnz ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
